org.eclipse.eclemma.doc/contexts_ui.xml

```diff
@@ -37,7 +37,7 @@
 		<topic label="Managing Coverage Sessions" href="pages/sessions.html"/>
 	</context>
 	<context id="session_export_context">
-		<description>Export a code coverage session as HTML report or EMMA data file.</description>
+		<description>Export code coverage sessions in different formats.</description>
```

Comment:
While we are on a subject of context help: on this line `coverage session`, while `coverage data` on lines 6 and 44 and `execution data` on line 27. However don't know if all those occurrences should be changed - documentation html-files contains all three term?